src/test/scala/test/TransformersSpec.scala

```diff
@@ -144,4 +146,22 @@ class TransformersSpec extends UnitSpec with LocalSparkContext {
     assert(df.schema == StructType(Array(StructField("data", JsonType, true))))
     assert(df.count == 2)
   }
+
+  "TwitterHashtagTransformer" should "should extract all the hashtags from the tweets resource" in {
+    val transform = new TwitterHashtagTransformer
+    val tweetsURI = getClass.getResource("/tweets").toURI
+    val rdd = sc.textFile(tweetsURI.toURL.toString).map(ByteUtils.utf8StringToBytes)
+
+    val tableName = "test"
+    val config = UserTransformConfig(
+      class_name = "test",
```

Comment:
Err, sorry, obviously it's used by the SparkInterface and is not relevant here.  But for clarity's sake, you should make it TwitterHashtagTransformer.